SROS2_Windows.md

```diff
@@ -14,7 +14,7 @@ Please follow [these instructions](https://docs.ros.org/en/rolling/Installation/
 
 To build the ROS2 code with security extensions, call:
```

Comment:
```suggestion
To build the ROS 2 code with security extensions, call:
```

(I know it is not directly connected with this PR, but while we are here we may as well fix it)